src/test/java/com/teragrep/pth10/evalTest.java

```diff
@@ -3134,46 +3547,29 @@ public void evalOperationLteTest() {
 
     // Tests EvalOperation after spath (spath makes all data into String)
     @Test
-    @DisabledIfSystemProperty(named="skipSparkTest", matches="true")
+    @DisabledIfSystemProperty(
+            named = "skipSparkTest",
+            matches = "true"
+    )
     public void evalAfterSpath_ComparisonTest() {
         String query = "index=index_A | spath path= json | eval a= json > 40";
         String schema = "StructType(StructField(_raw,StringType,true),StructField(_time,TimestampType,true),StructField(host,StringType,true),StructField(id,LongType,true),StructField(index,StringType,true),StructField(offset,LongType,true),StructField(partition,StringType,true),StructField(source,StringType,true),StructField(sourcetype,StringType,true),StructField(json,StringType,true),StructField(a,BooleanType,true))";
         String testFile = "src/test/resources/spath/spathTransformationTest_numeric2*.json";
 
         streamingTestUtil.performDPLTest(query, testFile, ds -> {
             assertEquals(schema, ds.schema().toString());
-            List<String> a = ds.select("a").orderBy("offset").collectAsList().stream().map(r -> r.getAs(0).toString()).collect(Collectors.toList());
-            List<String> expected = new ArrayList<>(Arrays.asList("false", "false", "false", "false", "true", "true", "true", "true", "true", "true"));
+            List<String> a = ds
+                    .select("a")
+                    .orderBy("offset")
+                    .collectAsList()
+                    .stream()
+                    .map(r -> r.getAs(0).toString())
+                    .collect(Collectors.toList());
+            List<String> expected = new ArrayList<>(
+                    Arrays.asList("false", "false", "false", "false", "true", "true", "true", "true", "true", "true")
+            );
 
             assertEquals(expected, a);
         });
     }
-
-    @Disabled(value="Should be changed to a dataframe test")
-	@Test
```

Comment:
were these implemented as dataframe tests?

Reply:
These already had dataframe tests with similar test queries, so they were just removed. The tests are called:
- parseEvalStrftimeCatalystTest()
- parseEvalStrptimeCatalystTest()
- parseEvalSplitCatalystTest()